src/lib/eliom_uri.shared.ml

```diff
@@ -18,8 +18,28 @@
  * Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA 02111-1307, USA.
 *)
 
-(*****************************************************************************)
 (* Building href *)
+
+let app_path = ref None
```

Comment:
It is only used on the client. It is not worth using Eliom syntax just for that, so I will see if I can find this info elsewhere (or else document it).